app/assets/stylesheets/pages/articles/article-list/_pagination.scss

```diff
@@ -0,0 +1,68 @@
+.pagination{
+	margin-top: 64px;
+	margin-bottom: 82px;
+
+	&__list-wrapper{
+		list-style: none;
+		padding-left: 0;
+	}
+
+	&__list-page{
+		display: inline-block;
+		margin-right: 10.2px;
+
+		&:hover .pagination__list-link{
+			background-color: $colorPrimary;
+			color: #fff;
+		}
+
+		&:first-child{
+			margin-left: 0;
+		}
+	}
+
+	&__list-link{
+		text-decoration: none;
+		color: $colorPrimary;
+		font-size: 22px;
+		padding: 2px 10px;
+		border-radius: 8px;
+		transition: all .3s ease-in-out;
+	}
+
+	&--first-page{
+		padding: 2px 13px;
```

Comment:
in which case you use it? because i saw it made the number one different to the others, why you did this?

Reply:
becuase I see when it is hovered, the size is not the same as the others,